aws/service.go

```diff
@@ -1155,6 +1156,22 @@ func StsService(ctx context.Context, d *plugin.QueryData) (*sts.STS, error) {
 	return svc, nil
 }
 
+// WafService returns the service connection for AWS WAF service
+func WafService(ctx context.Context, d *plugin.QueryData) (*waf.WAF, error) {
```

Comment:
```suggestion
func WAFService(ctx context.Context, d *plugin.QueryData) (*waf.WAF, error) {
```
This function is not correct, there is no return and bracket is also missing, this is not going to work.